### PR TITLE
Add Php Fpm runtime and Ruby versions

### PR DESCRIPTION
**Pull request by qianz2**
- Add PHP fpm 8.0 runtime
- Add Ruby 2.7.2 and 3.0.0

### CODE REVIEW

File: doc/runtimes/ruby.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ The latest version of *gem* is used to install dependencies.
 # Version support
 
 The Ruby project defines this [release schedule][]. Oryx supports all actively supported
-releases (2.5.8, 2.6.6, 2.7.1).
+releases (2.5.8, 2.6.6, 2.7.1, 2.7.2, 3.0.0).
```
Comment (gpcastro) [inline doc/runtimes/ruby.md:35]: do we want to maintain 2.7.1?

File: images/runtime/php-fpm/7.2/base.stretch.Dockerfile [hunk 2]
```diff
@@ -74,7 +74,7 @@ RUN docker-php-ext-configure pdo_odbc --with-pdo-odbc=unixODBC,/usr \
 #  - https://docs.microsoft.com/en-us/sql/connect/php/installation-tutorial-linux-mac
 #  - https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-microsoft-odbc-driver-for-sql-server
 RUN set -eux; \
-    if [[ $PHP_VERSION == 7.2.* || $PHP_VERSION == 7.3.* ]]; then \
+    if [[ $PHP_VERSION == 7.3.* ]]; then \
```
Comment (gpcastro) [inline images/runtime/php-fpm/7.2/base.stretch.Dockerfile:77]: is this generated from a template? Do weneed this check for 7.3 here if the dockerfile is in the 7.2 folder?

File: images/runtime/php-fpm/7.2/base.stretch.Dockerfile [hunk 3]
```diff
@@ -98,14 +98,6 @@ RUN { \
                 echo 'zend_extension=opcache'; \
     } > /usr/local/etc/php/conf.d/php.ini
 
-# Install the Microsoft SQL Server PDO driver on supported versions only.
-#  - https://docs.microsoft.com/en-us/sql/connect/php/installation-tutorial-linux-mac
-#  - https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-microsoft-odbc-driver-for-sql-server
-RUN set -eux; \
```
Comment (gpcastro) [inline images/runtime/php-fpm/7.2/base.stretch.Dockerfile:104]: just curious, why this is not needed anymore?

File: images/runtime/php-fpm/7.3/base.stretch.Dockerfile [hunk 4]
```diff
@@ -74,7 +74,7 @@ RUN docker-php-ext-configure pdo_odbc --with-pdo-odbc=unixODBC,/usr \
 #  - https://docs.microsoft.com/en-us/sql/connect/php/installation-tutorial-linux-mac
 #  - https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-microsoft-odbc-driver-for-sql-server
 RUN set -eux; \
-    if [[ $PHP_VERSION == 7.2.* || $PHP_VERSION == 7.3.* ]]; then \
+    if [[ $PHP_VERSION == 7.3.* ]]; then \
```
Comment (gpcastro) [inline images/runtime/php-fpm/7.3/base.stretch.Dockerfile:77]: likewise, do we need this check if the image is for version 7.3 (from the file path)?

**Review by gpcastro — APPROVED** [review]

File: doc/runtimes/ruby.md (hunk 1 above)
Comment (qianz2) [reply doc/runtimes/ruby.md:35]: Yes we still want to maintain it

File: images/runtime/php-fpm/7.2/base.stretch.Dockerfile (hunk 3 above)
Comment (qianz2) [reply images/runtime/php-fpm/7.2/base.stretch.Dockerfile:104]: When I try to run it, it shows that this feature was deprecated for 7.2.. so we're going to remove it from the template.

File: images/runtime/php-fpm/7.2/base.stretch.Dockerfile (hunk 2 above)
Comment (qianz2) [reply images/runtime/php-fpm/7.2/base.stretch.Dockerfile:77]: Yes it's from the template.base.Dockerfile, we still want to keep it same way as the template.

**Pull request merged**